Makefile

```diff
@@ -13,6 +13,7 @@ CONFIG_MANAGER_DIR ?= $(CONFIG_BUILD_DIR)/manager
 CONFIG_CRD_BASES_DIR ?= $(CONFIG_SRC_DIR)/crd/bases
 KUSTOMIZATION ?= default
 KUSTOMIZE_BUILD_DIR ?= $(CONFIG_BUILD_DIR)/$(KUSTOMIZATION)
+BUNDLE_DIR ?= bundle
```

Comment:
it looks like we are using this directory for temporary build resources, wonder if we could move it under `build`

Reply:
Yeah, I think that'll work, changed in d1c2c8a